### DISCUSSION TITLE
Testing Idena node: release candidate 0.29.2-rc0

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-go#978] [General]
We invite you to test the release candidate version of the Idena node: `0.29.2-rc0`
Please [download](https://github.com/idena-network/idena-go/releases/tag/v0.29.2-rc0) the node manually if you want to join the testing.

Changes:
- IPFS migration to version `0.12.2`

Please read it:
1. IPFS migration requires extra space on your hard drive. Make sure to have free space required for another copy of your `/ipfs` folder.
2. If the node can not start please delete `/ipfs` folder.
3. IPFS migration may take some time when the node starts for the first time.

We plan to release these changes soon unless negative feedback is received.

**Comment by rioda-org** [comment]
Works good for me. Node running at home. 650mb after 2days 9h.
